torch/nn/modules/module.py

```diff
@@ -340,14 +340,45 @@ def parameters(self, memo=None):
                 yield p
 
     def children(self):
-        """Returns an iterator over children modules."""
+        """Returns an iterator over immediate children modules."""
         memo = set()
         for module in self._modules.values():
             if module is not None and module not in memo:
                 memo.add(module)
                 yield module
 
+    def named_children(self):
+        """Returns an iterator over immediate children modules, yielding both
+        the name of the module as well as the module itself.
+
+        Example:
+            >>> for name, module in model.named_children():
+            >>>     if name in PRINT_LIST:
```

Comment:
can you replace `PRINT_LIST` with a more meaningful example like `{'conv1', 'fc5'}`?